guidelines/governance/sig-governance-guidelines.md

```diff
@@ -44,10 +44,11 @@ The technical lead MUST do the following:
 
 - Triage issues
 - Optimize software development lifecycle, process, and workflow
+- Resolve cross-project technical issues
 
 ## Project management
 
-Each SIG project MUST define the rationale, methodology, and public observability for
+The SIG MUST define the rationale, methodology, and public observability for
```

Comment:
I would stick with "each sig project", as it is better decentralized chain of responsibility in likely the event that projects under the same sig are run by different people and have different priorities and release dates